runtime-kanto/src/runtime/runtime_kanto.py

```diff
@@ -79,6 +83,19 @@ def adapt_feedercan_deployment_file():
         json.dump(data, f, indent=4)
 
 
+def adapt_mockservice_deployment_file():
```

Comment:
lets maybe have a generic function for adapting deployment files

Reply:
Will be tackled with the auto-generation of Kanto deployment files